ricardocosta/bank-ui
# feat(shell): Add navigation to shell application

## Summary





Add navigation to the sidebar in the shell application. We are using the router configuration to render the menu entries, and as such, we still don't have enough information to decide which icon to show.

Check #400 for more details on an API-based navigation.

<img width="1290" alt="image" src="https://user-images.githubusercontent.com/11922336/212545092-b761e88d-fd88-4ce4-93a8-7cc8aba2395f.png">


Closes #398

## Testing



- `yarn nx run shell:dev` to run the application locally.
- Open your browser in `http://localhost:3002/`
- The main page should show `Dashboard`
- Click the navigation entries and make sure the correct page is shown.

## Other Information

**Breaking Changes [YES / NO]** 
**Affected Component**
